src/coreclr/src/vm/reflectioninvocation.cpp

```diff
@@ -1322,11 +1316,7 @@ FCIMPL5(Object*, RuntimeMethodHandle::InvokeMethod,
     if (fConstructor)
     {
         // We have a special case for Strings...The object is returned...
```

Comment:
Is this comment still valid? Seems a but disconnected with the updated code.

Reply:
Yes, this comment is still valid. We will only get here for strings.